File: 2. Calculations/exercise11/python/currency_conversion.py
# Exercise 11 - Currency Conversion

# import ceiling function to round up numbers
from math import ceil

# Get input from users
euros = input('How many euros are you exchanging? ')
rate = input('What is the exchange rate? ')

# Convert input to correct type
euros = int(euros)
rate = float(rate)

# Perform calculations
dollars = ceil(euros * rate) / 100.0

# Format output
output_text = f'{euros} euros at an exchange rate of {rate}' \
              f' is {dollars} U.S. dollars.'

print(output_text)

File: 2. Calculations/exercise11/java/CurrencyConversion.java
import java.util.Scanner;

public class CurrencyConversion {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("How many euros are you exchanging? ");
        int euros = scanner.nextInt();

        System.out.print("What is the exchange rate? ");
        double rate = scanner.nextDouble();

        scanner.close();

        
        double dollars = Math.ceil(euros * rate) / 100.0;


        System.out.printf("%d euros at an exchange rate of %.2f is %.2f U.S. dollars.", euros, rate, dollars);
    }
}